packages/blockchain/test/clique.spec.ts

```diff
@@ -44,61 +44,61 @@ tape('Clique: Initialization', (t) => {
   }
 
   const B: Signer = {
-    address: new Address(Buffer.from('dc7bc81ddf67d037d7439f8e6ff12f3d2a100f71', 'hex')),
+    address: new Address(Buffer.from('6f62d8382bf2587361db73ceca28be91b2acb6df', 'hex')),
     privateKey: Buffer.from(
-      '86b0ff7b6cf70786f29f297c57562905ab0b6c32d69e177a46491e56da9e486e',
+      '2a6e9ad5a6a8e4f17149b8bc7128bf090566a11dbd63c30e5a0ee9f161309cd6',
       'hex'
     ),
     publicKey: Buffer.from(
-      'd3e3d2b722e325bfc085ff5638a112b4e7e88ff13f92fc7f6cfc14b5a25e8d1545a2f27d8537b96e8919949d5f8c139ae7fc81aea7cf7fe5d43d7faaa038e35b',
+      'ca0a55f6e81cb897aee6a1c390aa83435c41048faa0564b226cfc9f3df48b73e846377fb0fd606df073addc7bd851f22547afbbdd5c3b028c91399df802083a2',
       'hex'
     ),
   }
 
   const C: Signer = {
-    address: new Address(Buffer.from('8458f408106c4875c96679f3f556a511beabe138', 'hex')),
+    address: new Address(Buffer.from('83c30730d1972baa09765a1ac72a43db27fedce5', 'hex')),
```

Comment:
Lol, I also started fixing the tests yesterday but I went along with changing the order of the signers in the test cases. This is obviously the significantly better idea. 😎